specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -5398,6 +6419,94 @@
         }
       ],
       "x-ms-discriminator-value": "Custom"
+    },
+    "InferenceEndpointCreateRequest": {
+      "description": "The request to create an Inference Endpoint.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The description of the endpoint.",
+          "type": "string"
+        },
+        "properties": {
+          "description": "The endpoint properties dictionary. Properties are immutable.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "authMode":{
+          "type": "string",
+          "enum": [
+            "RBAC",
+            "AAD",
+            "Key"
+          ],
+          "x-ms-enum": {
+            "name": "AuthMode",
+            "modelAsString": true
+          },
+          "readOnly": true,
```

Comment:
readOnly in create request? what does that mean?

Reply:
that should be deleted for request